shells/pipes-shell-2/pipe.js

```diff
@@ -82,6 +83,12 @@ const populateDispatcher = (dispatcher, api, composerFactory, storage, context)
     },
     spawn: async (msg, tid, bus) => {
       return api.marshalSpawnArc(msg, tid, bus);
+    },
+    pec: async (msg, tid, bus) => {
```

Comment:
Move the next lines inside of pec-port.js, so it's just a function call here (as it is here, `pecPorts` is an implementation detail leaking out).